assets/javascripts/discourse/initializers/initialize-user-card-badges.js

```diff
@@ -1,17 +1,20 @@
-import computed from "discourse/lib/decorators";
 import { withPluginApi } from "discourse/lib/plugin-api";
 
 function modifyUserCardContents(api) {
-  api.modifyClass("component:user-card-contents", {
-    pluginId: "discourse-user-card-badges",
-
-    classNameBindings: ["hasCardBadgeImage"],
-
-    @computed("user.card_badge.image")
-    hasCardBadgeImage(image) {
-      return image && image.indexOf("fa-") !== 0;
-    },
-  });
+  api.modifyClass(
+    "component:user-card-contents",
+    (Superclass) =>
+      class extends Superclass {
+        get classNames() {
+          const result = [...super.classNames];
+          const image = this.user.card_badge?.image;
+          if (image && !image.includes("fa-")) {
```

Comment:
"technically correct" 😛 

```suggestion
          if (image && !image.startsWith("fa-")) {
```